src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java

```diff
@@ -238,7 +242,14 @@
     public void execute() throws MojoExecutionException, MojoFailureException {
         if (!skip) {
             boolean ansiRestore = Ansi.isEnabled();
-            log = new AnsiLogger(getLog(), useColorForLogging(), verbose, !settings.getInteractiveMode(), getLogPrefix());
+            File output = null;
+            if (outputFile != null) {
+                output = new File(outputFile);
+                if (output.exists()) {
+                    output.delete();
+                }
```

Comment:
I think we might not need to check whether outputFile exists and delete it. I think the default behavior of PrintWriter is to overwrite the file. What do you think?

Reply:
ohk, Thanks for clarification